### PR TITLE
added lists

### PR DESCRIPTION
**Pull request by MiryalaAkshitha**

### CODE REVIEW

File: components/list/list.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.list-container{
```
Comment (ramalingaa) [inline components/list/list.css:1]: try using responsive units instead absolute values

**Review by ramalingaa — APPROVED** [review]

File: components/list/list.css (hunk 1 above)
Comment (MiryalaAkshitha) [reply components/list/list.css:1]: changed thankyou

File: components/list/list.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        <li>Item one</li>
+        <li>Item two</li>
+        <li>Item three</li>
+    </ol>
+    <!-- stacked list -->
+    <ol class="stacked ol">
+        <li>Item one</li>
+        <ul>
+        <li>Item one 1</li>
+        <li>Item one 2</li>
```
Comment (romabulani) [inline components/list/list.html:41]: If possible, use prettier/beautify for formatting

**Review by romabulani — APPROVED** [review]
Rest all looks good to merge

**Pull request merged**